src/index.html

```diff
@@ -6,9 +6,31 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo" >
+        <img class="image" src="images\logo.png"
+          alt="logo of moyo-header">
+      </a>
+      <nav class="nav">
+        <ul>
+          <a class="link1 left is-active" href="#apple">APPLE</a>
+          <a class="link" href="#samsung">SAMSUNG</a>
+          <a class="link" href="#smartphones">SMARTPHONES</a>
+          <a class="link" data-qa="hover"
+            href="#laptops">LAPTOPS & COMPUTERS</a>
+          <a class="link" href="#gadgets">GADGETS</a>
+          <a class="link" href="#tablets">TABLETS</a>
+          <a class="link" href="#photo">PHOTO</a>
+          <a class="link right " href="#video">VIDEO</a>
+        </ul>
```

Comment:
1. please remember about list structure, `<ul>` should have children `<li>` elements,
2. remember about BEM block__element -> if you have links inside block nav, so class should be `.nav__link`
3.  left, right classes are not needed